docs/src/developing/on-chain-programs/faq.md

```diff
@@ -6,7 +6,7 @@ When writing or interacting with Solana programs, there are common questions or
 challenges that often come up. Below are resources to help answer these
 questions.
 
-If not addressed here, the Solana [#developers](https://discord.gg/RxeGBH)
+If not addressed here, ask on [StackOverflow](https://stackoverflow.com/questions/tagged/solana) with the `solana` tag or check out the Solana [#developer-support](https://discord.gg/RxeGBH)
 Discord channel is a great resource.
```

Comment:
Mind removing this line as well

Reply:
Removed in 65934cc8270fcb39052e31542bde09f9575b4e54